src/questions/author-linkedin.js

```diff
@@ -0,0 +1,8 @@
+const { cleanSocialNetworkUsername } = require('../utils')
+
+module.exports = () => ({
+  type: 'input',
+  message: '🕴  LinkedIn username (use empty value to skip)',
```

Comment:
```suggestion
  message: '💼  LinkedIn username (use empty value to skip)',
```

Reply:
> Nice job @anku255! What do you think of using 💼 ?

Yes, I also thought about using this emoji. 👍